package.json

```diff
@@ -4,8 +4,10 @@
   "description": "file-loader that stores a map of original file names to file names with hashes in the compilation stats",
   "main": "index.js",
   "scripts": {
-    "test": "env NODE_ENV=test mocha --require ./test/power_assert_loader.js test/basic.js test/integration.js --watch",
-    "test-ci": "env NODE_ENV=test mocha --require ./test/power_assert_loader.js test/basic.js test/integration.js"
+    "lint": "standard",
+    "lint-fix": "standard --fix",
+    "test": "npm run lint && env NODE_ENV=test mocha --require ./test/power_assert_loader.js test/basic.js test/integration.js --watch",
```

Comment:
This will only run the linter one time, and then watch the unit tests.  If we'd prefer, I can set up [chokidar](https://github.com/paulmillr/chokidar) to run both tasks when files change.